ios/HackerNews/HNApp.swift

```diff
@@ -35,9 +36,36 @@ struct HackerNewsApp: App {
           print("Failed to submit feedback: \(error)")
         }
       }
+      
       options.enableAppHangTrackingV2 = true
       options.sessionReplay.onErrorSampleRate = 1.0
+      options.sendDefaultPii = true
+      
+#if DEBUG
+      options.environment = "development"
+      options.sessionReplay.sessionSampleRate = 1.0
+      options.tracesSampleRate = 1
+//      options.debug = true
```

Comment:
```suggestion
      options.debug = true
```
Not sure if you meant to comment this for debug builds

Reply:
Yea I left this commented out for now since it was way noisier than I thought it would be.